webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudConnectorButton/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { CloudConnectorButton } from './CloudConnectorButton';
+import { configureCloudConnector } from './CloudConnectorActions';
+import { selectStatus, selectJobLink } from './CloudConnectorSelectors';
+
+const ConnectedCloudConnectorButton = () => {
+  const status = useSelector(selectStatus);
+  const jobLink = useSelector(selectJobLink);
+  const dispatch = useDispatch();
+  return (
+    <CloudConnectorButton
+      status={status}
+      onClick={() => dispatch(configureCloudConnector)}
```

Comment:
for now, it doesn't fire the `configureCloudConnector` on click

Reply:
Changed to `configureCloudConnector()`